.github/workflows/build.yml

```diff
@@ -16,7 +16,7 @@ jobs:
       - uses: arduino/setup-protoc@v1
       - run: go install google.golang.org/protobuf/cmd/protoc-gen-go@v1.27.1
       - run: go mod download
-      - run: make generate
-      - run: make format
+      - run: make
       - run: git status
+      - run: git diff
       - run: git diff --quiet *.proto codec/*
```

Comment:
Do we need these two diffs here or just a single one would do?

Reply:
single is fine but make it easier to debug when the workflow fail. Want  me to remove the first git diff ?

Reply:
No, let's keep it if this is intentional.